site/content/docs/5.3/layout/css-grid.md

```diff
@@ -227,13 +227,21 @@ Adding more rows and changing the placement of columns:
 
 ### Gaps
 
-Change the vertical gaps only by modifying the `row-gap`. Note that we use `gap` on `.grid`s, but `row-gap` and `column-gap` can be modified as needed.
+Change the vertical gaps only by modifying the `row-gap`. Note that we use `gap` on `.grid`s, but `row-gap` and `column-gap` can be modified as needed using either CSS properties, or CSS variables, or our [gap utilities]({{< docsref "/utilities/spacing#gap" >}}).
```

Comment:
Having a new example and mentioning that we can use the gap utilities can be interesting. However, there's something weird in the order here. The first sentence mentions that we can modify the `row-gap`, and then, the first example doesn't modify the `row-gap`. Technically, and visually, `.gap-3` modifies `gap` and not `row-gap`.
I'd say that there's something to enhance with this modification.